pom.xml

```diff
@@ -235,6 +234,41 @@
 
   <!-- This profile is used to allow github action to build branches. The github action is used for sonar analysis -->
   <profiles>
+    <profile>
```

Comment:
why we need to define the profile here as it is already in maven-parent-pom ?

Reply:
It's to declare the `AspectJ custom Annotation processing` defined in module `portal-component-common` to be used when weaving/processing the class file, here is the main declaration needed for this profile (**<aspectLibraries>** not <dependency>):

```xml
              <aspectLibraries>
                <aspectLibrary>
                  <groupId>io.meeds.portal</groupId>
                  <artifactId>portal.component.common</artifactId>
                </aspectLibrary>
              </aspectLibraries>

```